### PR TITLE
Update strings.js

### PR DESCRIPTION
**Pull request by christophrrb**
Cleaned up one or two English and German translations and added Spain Spanish and some missing German translations.

### CODE REVIEW

**Comment by pcottle** [comment]
Wow awesome @christophrrb ! 🎉  How multilingual of you :D

File: src/js/intl/strings.js [hunk 1]
```diff
@@ -413,6 +442,8 @@ exports.strings = {
     'zh_CN': '你所指定的参数不兼容或者不准确',
     'zh_TW': '您指定了不相容或錯誤的選項',
     'es_AR': 'Las opciones que especificaste son incompatibles o incorrectas',
+    'es_ES': 'Las opciones que especificaste son incompatibles o incorrectas.',
+    'es_AR': 'No es necesario hacer add a los archivos en esta demo',
```
Comment (pcottle) [inline src/js/intl/strings.js]: whoops found a few dupe keys, this is preventing pushing the site. ill try to fix this

File: src/js/intl/strings.js [hunk 2]
```diff
@@ -1077,6 +1159,7 @@ exports.strings = {
     'zh_CN': '您正在进行关卡构建中，这里有多种形式的帮助，请选择 "help general" (一般帮助)或 "help builder" (关卡构建帮助)',
     'zh_TW': '您正在進行關卡構建中，這裡有多種不同型式的幫助，請選擇 "help general" （一般幫助）或 "help builder" （製造關卡的幫助）',
     'es_AR': 'Estás en el constructor de niveles, por lo que hay varios tipos de ayuda. Elegí entre "help general" para ayuda sobre Learn GitBranching y "help builder" para ayuda sobre el constructor de niveles',
+    'es_AR': 'Estás en un constructor de niveles, por lo que hay varios tipos de ayuda. Elige "help general" para ayuda sobre Learn GitBranching o "help builder" para ayuda sobre el constructor de niveles',
```
Comment (pcottle) [inline src/js/intl/strings.js]: im assuming this is es_ES ?

Comment (christophrrb) [reply src/js/intl/strings.js]: Woops! Yes. Thanks for catching that. I should have looked over it better.

**Pull request merged**